faq.md

```diff
@@ -3,3 +3,27 @@ layout: page
 title: FAQ
 permalink: /faq
 ---
+# How can I help?
+
+If you have a service like this for people already, get in contact with us at [contact@vacfind.org][1]!
+We are commited to bringing easy to access information such as where to get a vaccine to the general public.
+Have some spare time on your hands?
+[Email us][1]!
+
+# How can Hospitals help?
+
+Hospitals and clincs administering vaccines are our end targets.
+We can add your data to our site and help bring people to you.
+It will also relieve your databases and phone calls as we can provide this directly to your customers.
+All we need is up to date information in whatever form you find easiest.
```

Comment:
1. this doesn't seem consistent/aligned with the info in #4 regarding the ways we help people since we didn't mention that we were directly keeping track of vaccination sites ourselves. IMO we should work with GISCorps to help them update their data from hospital availability both to benefit others using that dataset and so that we don't need to spend time re-doing it

Reply:
I changed the wording here to be "retrieve databases."